components/notability/commons/notability_checker.lua

```diff
@@ -231,7 +231,7 @@ end
 
 function NotabilityChecker._calculateDateLoss(date)
 	local timestamp = _lang:formatDate('U', date)
-	local differenceSeconds = _NOW - timestamp
+	local differenceSeconds = math.max(_NOW - timestamp, 0) -- No dates in the future
```

Comment:
The comment basically just describes the code, not the "why".

Reply:
Updated